gateway/src/main.rs

```diff
@@ -63,34 +55,70 @@ async fn main() -> io::Result<()> {
         // LD_LIBRARY_PATH env set in build.rs.
         .extension("ulid0");
 
+    info!("Connecting and migrating db...");
     let db = SqlitePool::connect_with(sqlite_options).await.unwrap();
     MIGRATIONS.run(&db).await.unwrap();
 
     match args.command {
         Commands::Start(start_args) => start(db, args.state, posthog_client, start_args).await,
+        Commands::Sync(sync_args) => sync_permit_projects(db, sync_args).await,
+    }
+}
+
+async fn sync_permit_projects(db: SqlitePool, args: SyncArgs) {
+    let client = permit::Client::new(
+        args.permit.permit_api_uri.to_string(),
+        args.permit.permit_pdp_uri.to_string(),
+        "default".to_owned(),
+        args.permit.permit_env,
+        args.permit.permit_api_key,
+    );
+
+    let projects: Vec<(String, String)> =
+        sqlx::query_as("SELECT user_id, project_id FROM projects")
+            .fetch_all(&db)
+            .await
+            .unwrap();
+
+    for (uid, pid) in projects {
+        match client.get_user_projects(&uid).await {
+            Ok(projs) => {
+                if !projs
+                    .iter()
+                    .any(|p| p.resource.as_ref().unwrap().key == pid)
```

Comment:
Wait, we get all the projects from permit to check if just one is correct in each loop? Maybe we should as permit for just the one project. Or group the DB results first.